lib/runtimeSettings.js

```diff
@@ -109,12 +134,16 @@ module.exports = {
             token: '${settings.projectToken}'
         }
     },
+    ${nodesDir}
     editorTheme: {
+        ${projectSettings.theme}
```

Comment:
`projectSettings.theme` defaults to the value `forge-light` - the bare name of the theme to use.

If the platform provided settings object includes a `theme` property then `projectSettings.theme` gets set to `theme: '${settings.settings.theme}'` - the line to include in the settings file.

When it gets inserts into the settings.js template, it assumes its is the full line, not just the name of the theme.

This means, if the platform settings object doesn't include the theme name for any reason, then the settings.js template will have the bare name of the theme inserted - creating an invalid file.

Reply:
I've read this comment a few times (I must be tired Nick). I'm not certain I follow.

Reply:
More simply put: if `settings.settings.theme` is not set, then the settings.js file will end up looking like this:

```
module.exports = {
    ...
    editorTheme: {
        forge-light
        page: {
            ...
```

If `settings.settings.theme` is set to `foo`, then the settings.js file will end up looking like this:

```
module.exports = {
    ...
    editorTheme: {
        theme: 'foo',
        page: {
            ...
```

Reply:
I've pushed the fix for this issue.